File: Extras/triangulo.py
# -*- coding: utf-8 -*-

t = 0
while t < 1 or t > 80:
    t = int(raw_input("Qué tamaño de triángulo quieres?: "))

c = raw_input("Qué caractér quieres que use?: ")

for i in range(t):
    print c * i

for j in range(t, 0, -1):
    print c * j

File: Extras/triangulo.cpp
#include <iostream>

using namespace std;

int main(int argc, char* argv[]) {
	int t,cont=0;
	char c;

	do {
		cout << "Qué tamaño de triángulo quieres?: ";
		cin >> t;
	} while (t > 80 or t < 1);
	cout << "Qué caractér quieres que use?: ";
	cin >> c;


	for (int i=0; i < t; i++) {
		for (int j=0; j < i; j++) {
			cout << c;
		}
		cout << endl;
	}

	for (int k=0; k < t; k++) {
		for (int m=t; m > k; m--) {
			cout << c;
		}
		cout << endl;
	}
	
	return 0;
}